backend/.eslintrc.json

```diff
@@ -20,7 +21,8 @@
         "prettier/prettier": [
             "error",
             {
-                "endOfLine": "auto"
+                "endOfLine": "auto",
+                "varsIgnorePattern": "should|expect"
```

Comment:
check if there are better alternatives to ignoring this pattern

Reply:
I checked but I didn't find any other alternative to this but if you find any then feel free to make your own changes.